### ISSUE TITLE
Car Play - Turn signal not updated

### ISSUE THREAD

**Issue by FieteO** [organicmaps/organicmaps#7589]
⚠ Have you searched for similar, already existing issues?
Yes

**Describe the issue**
Please write a clear and concise description of the issue here.
While using the Apple Car Play mode, the turn signal is not updated. This happened right after starting the navigation.

After some time, it is updates correctly

**Steps to reproduce**

**Expected behaviour**
The turn signal is in sync with the navigation Ehen starting the navigation.

**Screenshots**
If applicable, add screenshots or screen recordings to help explain your problem.
[
![IMG_20240314_103309](https://github.com/organicmaps/organicmaps/assets/41323592/4f0c2107-d75e-49e7-b52f-0ea80e701771)
](url)

**System information:**
 - Operating system and its version: [iOS 12, Android 10, Ubuntu 22, MacOS Big Sur, etc.]
 - iOS 17.3
 - Organic Maps version: [you can find it by tapping the button with the green Organic Maps logo]
 - 2024.03.05
 - Device Model: [e.g. iPhone 6, Samsung S22]
Iphone 14

**Additional context**
Please add any other context or comments here that may be useful.

**Comment by biodranik** [comment]
Is arrow position and other info (distance, time) updated properly at that time?

**Comment by FieteO** [comment]
Yes I think so. I will double check once I'll be driving next time
